core/test/martian/schema_test.cljc

```diff
@@ -400,3 +401,34 @@
 
     (is (= (s/maybe {(s/optional-key :b) (s/maybe {(s/optional-key :a) s/Any})})
            schema))))
+
+
+
+(deftest resolve-ref-object-test
+  (let [params       {:idParam {:name "id" :in "query"}}
+        swagger-spec {:definitions params}
+        openapi-spec {:components {:parameters params}}
+        #?(:cljs ExceptionInfo) #?(:cljs js/Error)]
```

Comment:
This is a hack to work with ClojureScript..
we might want to change it to some test-helper like this:
https://gist.github.com/metametadata/69cae3488d891f5e23d9

Reply:
I tend to just do this at the top level:
```
#?(:cljs (def ExceptionInfo js/Error))
```